ngraph/frontend/onnx_import/src/onnx.cpp

```diff
@@ -73,6 +73,28 @@ namespace ngraph
 
             } // namespace error
 
+            static const std::vector<std::string> legacy_ops_to_fixup = {
+                "FakeQuantize", "DetectionOutput", "Normalize", "PriorBox"};
+
+            // There are some models with custom OPs (list above) that has the default domain set.
+            // So in order to load the models, we need overwrite the OPs' domain to the one they're
+            // registered
+            void fixup_legacy_operators(ONNX_NAMESPACE::GraphProto* graph_proto)
+            {
+                for (auto& node : *graph_proto->mutable_node())
+                {
+                    auto it = std::find(
+                        legacy_ops_to_fixup.begin(), legacy_ops_to_fixup.end(), node.op_type());
+                    if (it != legacy_ops_to_fixup.end())
+                    {
+                        if (!node.has_domain() || node.domain() == "ai.onnx")
```

Comment:
Can we also check the ONNX opset version?

Reply:
can you elaborate more? What should I compare the version to?

Reply:
Discussed it offline.

Reply:
If we don't have a good solution to prevent applying the domain fix after some pre-specified opset version, then OK, let commit it as-is. @mateusztabaka, please confirm that we don't have a good solution for that.